cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceConfig.java

```diff
@@ -28,14 +28,22 @@
   @SerializedName("scheduler.queue.name")
   private final String schedulerQueueName;
 
-  public NamespaceConfig(String schedulerQueueName) {
+  @SerializedName("hbase.namespace")
```

Comment:
I think @bdmogal is suggesting to simply remove line 31. That shouldn't cause any compilation issues.